server/lib/pbench/__init__.py

```diff
@@ -198,10 +198,10 @@ def __init__(self, cfg_name):
             if not os.path.isdir(self.LIBDIR): raise BadConfig("Bad LIBDIR={}".format(self.LIBDIR))
 
             self.ARCHIVE = self.conf.get("pbench-server", "pbench-archive-dir")
-            self.INCOMING = self.conf.get("pbench-server", "pbench-incoming-dir")
+            self.INCOMING = os.path.join(self.TOP, "public_html/incoming")
```

Comment:
The INCOMING, RESULTS, USERS should be moved to an else clause. No slashes in the paths.